dracut/anaconda-lib.sh

```diff
@@ -434,3 +434,14 @@ wait_for_updates() {
 wait_for_dd() {
     echo "[ -e /tmp/dd.done ]" > "$hookdir/initqueue/finished/dd.sh"
 }
+
+wait_for_disks() {
+    # Allow up to 'inst.wait_for_disks' seconds for disks to be enumerated and
+    # related udev rules to execute (defaults to 5 seconds, 0 disables the
+    # feature)
+    finished_hook="$hookdir/initqueue/finished/wait_for_disks.sh"
+    [ -e "$finished_hook" ] && return
+    DISKS_WAIT_DELAY=$(getargnum 5 0 10000 inst.wait_for_disks)
+    DISKS_WAIT_RETRIES=$(($DISKS_WAIT_DELAY * 2))
```

Comment:
Any idea what is wrong here?
~~~
dracut/anaconda-lib.sh:445:27: note: $/${} is unnecessary on arithmetic variables. [SC2004]
dracut/anaconda-lib.sh:446:54: note: Double quote to prevent globbing and word splitting. [SC2086]
~~~